bare-metal/elastic-metal/api-cli/elastic-metal-with-api.mdx

```diff
@@ -18,13 +18,12 @@ Our [Elastic Metal server](https://www.scaleway.com/en/elastic-metal/) offer pro
 
 The new deployment in one command feature allows you to speed up the delivery of your servers, as you do not have to wait for it to be delivered to install the server. You can do both in a single API call, and your server is up and running in less than 15 minutes.
 
-<Macro id="iam-requirements" />
+## Requirements
```

Comment:
This PR seems to be missing the new macro and "Before you start" formulation

Reply:
There are too many bugs in this PR, so I'll close it and create a new one. :lolcry:
Thanks!